src/settings.js

```diff
@@ -735,22 +735,22 @@ var ASYNCIFY_STACK_SIZE = 4096;
 // builds, etc.). You can inspect the wasm binary to look for the actual names,
```

Comment:
Github won't let me comment on it because it's not part of the diff, but should like 697 above actually be "do an async operation"?